docs/content/Select.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Select
+description: Use a select input when a user needs to select one option from a long list
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Select.tsx
+storybook: '/react/storybook?path=/story/forms-select--default'
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
```

Comment:
Don't think we need to import this anymore as it's global

Reply:
That's what I thought too, but if I don't import it, it doesn't render.

Reply:
Could your `/docs` node_modules need updating?